inc/ticket.class.php

```diff
@@ -5238,11 +5238,16 @@ static function showListForItem(CommonDBTM $item) {
             // you can only see your tickets
             if (!Session::haveRight(self::$rightname, self::READALL)) {
                $restrict .= " AND (`glpi_tickets`.`users_id_recipient` = '".Session::getLoginUserID()."'
-                                   OR (`glpi_tickets_users`.`tickets_id` = '".$item->getID()."'
+                                   OR (`glpi_tickets_users`.`tickets_id` = `glpi_tickets`.`id`
                                        AND `glpi_tickets_users`.`users_id`
                                             = '".Session::getLoginUserID()."')
                                    OR `glpi_groups_tickets`.`groups_id` IN (".implode(",",$_SESSION['glpigroups'])."))";
             }
+
+            if (Session::haveRight(self::$rightname, self::ASSIGN)) {
```

Comment:
If you have right to assign tickets, you see new tickets in "tickets list page". (not assigned)
So, it's for this reason, i propose to add : 

if (Session::haveRight(self::$rightname, self::ASSIGN)){
$restrict .= "OR (glpi_tickets.status=".self::INCOMING.")";}